src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int OPERATION_PART = 0;
+    private static final int VALUE_PART = 1;
+    private static final int FIRST_VALUE = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
```

Comment:
Create separate methods with these logic
read from file
generate report (calculcate report) 
write to file 


You don't need to create File file = new File(fromFileName); to implement the method